.github/workflows/python-package.yml

```diff
@@ -34,7 +34,7 @@ jobs:
     - name: Lint with Black
       run: |
         # stop the build if there are Python syntax errors or undefined names
-        python -m black . --check
+        python -m black .
```

Comment:
`--check` is required for lint, otherwise it just formats the files but doesn't throw any legit errors. You just need to specify the python folders and files, so I think the right way would be: `python -m black --check src test setup.py`

Reply:
The original "." checks all files under the current directory, but seems it does not work as before. The format check works locally but fails here. Please check the last run, files specification as mentioned does not help here :(

Reply:
Sounds cool, thank you!